tests/utils.py

```diff
@@ -39,10 +39,14 @@ def resolve_block_bytes(block_bytes):
 def gen_new_double_spending(manager: HathorManager, *, use_same_parents: bool = False,
                             tx: Optional[Transaction] = None) -> Transaction:
     if tx is None:
-        tx_interval = random.choice(list(manager.tx_storage.get_tx_tips()))
-        _tx = manager.tx_storage.get_transaction(tx_interval.data)
-        assert isinstance(_tx, Transaction)
-        tx = _tx
+        possible_parents = manager.get_new_tx_parents()
```

Comment:
I think the name `possible_parents` is misleading. I think a better name is `tx_candidates`. What do you think?

Reply:
Seems better, I'll change it.

Reply:
Seems better, I'll change it.